user_docs/en/changes.md

```diff
@@ -85,6 +85,7 @@ Please responsibly disclose security issues following NVDA's [security policy](h
 * Added a general setting to prevent the display turning off during say all or reading with braille.
 This option is enabled by default, but may result in increased battery depletion. (#17649, @LeonarddeR)
 * NVDA is now translated into Bosnian. (#17953)
+* In Adobe Acrobat, NVDA can now read and interact with math equations in PDF documents generated by recent versions of Microsoft Word. (#18056)
```

Comment:
Could this item please be moved to be under "Support for math in PDFs has been added."

Reply:
I'm not quite sure how to word that, as that talks about it only being available for Foxit Reader & Foxit Editor, which is not a tall true for my change.
I think it might be better to keep them separate and perhaps consider reworking the top one to specifically say Foxit rather than just pdf.

Reply:
Fair enough